cmd/ci-secret-bootstrap/main_test.go

```diff
@@ -1733,3 +1808,148 @@ func equal(t *testing.T, what string, expected, actual interface{}) {
 		t.Errorf("%s differs from expected:\n%s", what, cmp.Diff(expected, actual))
 	}
 }
+
+func TestConstructDockerConfigJSON(t *testing.T) {
+	testCases := []struct {
+		id                   string
+		bwClient             bitwarden.Client
+		dockerConfigJSONData []secretbootstrap.DockerConfigJSONData
+		expectedJSON         []byte
+		expectedError        string
+	}{
+		{
+			id: "happy case",
+			dockerConfigJSONData: []secretbootstrap.DockerConfigJSONData{
+				{
+					BWItem:    "item-name-1",
+					EntryName: "entryName",
+					Auth:      "auth",
+					Email:     "email",
+				},
+			},
+			bwClient: bitwarden.NewFakeClient(
+				[]bitwarden.Item{
+					{
+						ID:   "1",
+						Name: "item-name-1",
+						Fields: []bitwarden.Field{
+							{
+								Name:  "entryName",
+								Value: "quay.io",
+							},
+							{
+								Name:  "auth",
+								Value: "123456789",
+							},
+							{
+								Name:  "email",
+								Value: "test@test.com",
+							},
+						},
+					},
+				}, nil),
+			expectedJSON: []byte("{\"auths\":{\"quay.io\":{\"auth\":\"123456789\",\"email\":\"test@test.com\"}}}"),
```

Comment:
Maybe use
```
`{"auths":[...}`
```

So you don't have to escape all the quotes and this becomes more readable?